tests/TestGui/log_header.tcl

```diff
@@ -0,0 +1,51 @@
+gui_start
+
+create_design log_header
+
+analyze
```

Comment:
This is a good start. I don't see a header in the synthesis.rpt file tough with aes_decrypt example. Can you please confirm that we have a header in synthesis too?

Reply:
Ok, I had to re-run it for some reason, I was able to see it. Thanks!